mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -793,6 +793,14 @@ static void correctConvParameters() {
   else if (inputLayoutValue.size() == 4)
     inputLayout = "g" + inputLayoutValue;
 
+  // +++pf:  update old key names.
```

Comment:
rocmlir-gen will accept either old or new keys, for now.

Reply:
This we might as well keep around for ergonomics